modules/local/eggnog.nf

```diff
@@ -44,6 +42,8 @@ process EGGNOG_MAPPER {
         --data_dir ${eggnog_db_dir} \
         --no_file_comments \
         --cpu ${task.cpus} \
+        --tax_scope 'prokaryota_broad' \
```

Comment:
Should we consider the lineage here?

Reply:
That's very tricky to do on diverse taxa some of which might be unknown. I think having prokaryota_broad as a scope is a good compromise overall - better than no tax scope but shouldn't create issues for unknown taxonomies.

Reply:
and `archea`?